pset-functions.js

```diff
@@ -208,6 +307,23 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const firstWord = "loop"
+const secondWord = "rune"
+const thirdWord = "apple"
+
+const checkString = function(word) {
+    for (let i = 0; i < word.length; i++) {
+      if (word.indexOf(word[i]) !== word.lastIndexOf(word[i])) {
+        return true
+      }
+    }
+  return false
+}
+
+console.log(checkString(firstWord))
+console.log(checkString(secondWord))
+console.log(checkString(thirdWord))
```

Comment:
@rlou88 Good Job here with this solution. The function loops through the letters of the word and nice job using the method `indexOf` and `lastIndexOf` to compare the repeated letters and return true or false

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)